packages/salesforce-adapter/src/transformers/xml_transformer.ts

```
@@ -346,9 +357,12 @@ export const fromRetrieveResult = async (
   const fromZip = async (zip: JSZip, file: FileProperties): Promise<MetadataValues | undefined> => {
     // extract metadata values
     const fileNameToValuesBuffer = await extractFileNameToData(zip, file.fileName,
-      typesWithMetaFile.has(file.type) || isComplexType(file.type), isComplexType(file.type))
+      typesWithMetaFile.has(file.type) || isComplexType(file.type), isComplexType(file.type),
+      file.namespacePrefix)
     if (Object.values(fileNameToValuesBuffer).length !== 1) {
-      log.warn(`Expected to retrieve only single values file for ${file.type}`)
+      if (file.fullName !== UNFILED_PUBLIC_FOLDER) {
```

why are we excluding this specifically?
is this true for all types? is it impossible to have a non folder type with this name?

---

When we try to retrieve a certain type that has folders, we receive also the `UNFILED_PUBLIC_FOLDER`. This is not really a folder in SF, meaning that it has no metadata in the retrieve result. The reason I excluded it is just to avoid printing an irrelevant warning to the log file.
It's not possible to have this name for other types as it contains the `$` char.

---

if it really doesn't exist and we are hard-coding this knowledge into our adapter, maybe it would make more sense to filter it out earlier?
if we know it isn't going to work, I think it might if we don't try to fetch it instead of suppressing the error we get when we realize it doesn't exist

---

We don't try to fetch it explicitely, it just arrived in the retrieve result.